table/tables/tables.go

```diff
@@ -84,6 +84,10 @@ func TableFromMeta(alloc autoid.Allocator, tblInfo *model.TableInfo) (table.Tabl
 			if err != nil {
```

Comment:
If we store parsed expr in tikv, we need serialize/deserialize ast structs. There will be too much code to write.